client/src/components/graphs/OrganizationalChart.js

```diff
@@ -294,16 +316,26 @@ const OrganizationalChart = ({
           : d.data.longName
       )
 
+    // Highlight all leaders
     const headG = nodeSelect.selectAll("g.head").data(
-      d => sortPositions(d.data.positions, Math.min(1, personnelDepth)) || [],
+      d => {
+        updateLeaderCount(d.data.uuid, d.data.positions)
```

Comment:
Instead of using `updateLeaderCount()` and `leaderCountMap`, perhaps simply filter the positions for leaders here (and for non-leaders below).